Jenkinsfile

```diff
@@ -49,22 +49,22 @@ pipeline {
                 test_flags = "${KATPACKAGE}"
             }
 
-             parallel {
-                stage ('py27') {
-                    steps {
+             steps {
+
                         echo "Running nosetests on Python 2.7"
                         sh 'python2 -m pip install -U .'
-                        sh "python2 setup.py nosetests --with-xunitmp --xunitmp-file=nosetests_py27.xml"
-                    }
-                }
+                        sh 'python2 -m coverage run --source="${KATPACKAGE}" -m nose --with-xunitmp --xunitmp-file=nosetests_py27.xml'
+                        sh 'python2 -m coverage xml -o coverage_27.xml'
+                        sh 'python2 -m coverage report -m --skip-covered'
+
+                        //sh 'unlink /usr/bin/python && ln -s /usr/bin/python3.6 /usr/bin/python'
```

Comment:
Will we need this line of code in future? else remove it.

Reply:
I wanted to run the test with those changes only and see what happens. 
The one problem we do have tho is that when the python3 tests run and if scripts are executed  with ```#!/usr/bin/env python```  then it will return python2 and not python3.

But I plan to remove it after I test python3 only on a node with both python2 and python3

Reply:
Cool.